recipes/audi/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.2" %}
+
+package:
+  name: audi
+  version: {{ version }}
+
+source:
+  url: https://github.com/darioizzo/audi/archive/v{{ version }}.tar.gz 
+  sha256: efd4bf289836309e8ce6a71dcc0d46f3df0444e276a812cbf2761b0a39fd019d
+
+build:
+  number: 0
+  skip: true  # [win]
+  
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - boost-cpp 1.64.0
+    - eigen
+    - piranha 0.10
+
+  run:
+    - boost-cpp 1.64.0
```

Comment:
Can you change this to 1.66.0?

Reply:
piranha 0.10 is pinned to 1.64, unfortunately. @bluescarni is working on a code refactoring of piranha, in the meantime I have to use v0.10

Reply:
Is it possible to rebuild piranha 0.10 with 1.66?

Reply:
I think it should be, @bluescarni any reason not to?